globus_sdk/transfer/paging.py

```diff
@@ -16,24 +16,27 @@ class PaginatedResource(GlobusResponse, six.Iterator):
 
```

Comment:
Somewhere in this helptext I think we should re-emphasize that the iterator can only be walked once and that each iteration makes a network-call.